src/components/Footer/styles.css

```diff
@@ -0,0 +1,521 @@
+@import url('https://rsms.me/inter/inter.css');
+html { font-family: 'Inter', sans-serif; }
+@supports (font-variation-settings: normal) {
+  html { font-family: 'Inter var', sans-serif; 
+    background: #F2F2F2;}
+}
+
+footer {
+    background: #FFFFFF;
+}
+
+#footer-component {
+    width: 100%;
+    display: grid;
+    justify-content: center;
+    text-align: center;
+    background: #FFFFFF;
```

Comment:
A good idea for the next ones would be to use css variables for colors. Also, you can just type `white` instead of #ffffff